README.md

```diff
@@ -96,10 +96,31 @@ Ensure the following elements are present in your `manifest.yml`. The `run_migra
 To seed fund & round data to db
 
 ```
-docker exec -ti <fund_store_container_id> python -m scripts.load_cof_r2
-docker exec -ti <fund_store_container_id> python -m scripts.load_cof_r3w1
-docker exec -ti <fund_store_container_id> python -m scripts.load_ns_r2
+docker exec -ti $(docker ps -qf "name=fund-store") python -m scripts.load_cof_r2
+docker exec -ti $(docker ps -qf "name=fund-store") python -m scripts.load_cof_r3w1
+docker exec -ti $(docker ps -qf "name=fund-store") python -m scripts.load_cof_r3w2
+docker exec -ti $(docker ps -qf "name=fund-store") python -m scripts.load_ns_r2
```

Comment:
that's cool implementation. just need to be sure only one container has "fund-store" like name